src/Tribe/Commerce/Warnings.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Conditional Warnings on Ticket Features
+ *
+ * Class Tribe__Tickets__Commerce__Warnings
+ *
+ * @since TBD
+ *
+ */
+class Tribe__Tickets__Commerce__Warnings {
+
+	public function hook() {
+		add_action( 'tribe_events_tickets_new_ticket_buttons', [ $this, 'recurring_event_warning_msg' ] );
+	}
+
+	/**
+	 * Create Recurring Event Warning Message Label
+	 *
+	 * @since 4.6
+	 *
+	 * @param int $post_id Post ID.
+	 *
+	 */
+	public function recurring_event_warning_msg( $post_id ) {
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 9b45db9aeabf5b4c08d0077bd0310bdf</sub>